docs/maui/behaviors/user-stopped-typing-behavior.md

```diff
@@ -0,0 +1,106 @@
+---
+title: UserStoppedTypingBehavior - .NET MAUI Community Toolkit
+author: cliffagius
+description: "The UserStoppedTypingBehavior is a Behavior that allows the user to trigger an action when a user has stopped data input an Entry. Examples of its usage include triggering a search when a user has stopped entering their search query."
+ms.date: 04/13/2022
+---
+
+# UserStoppedTypingBehavior
+
+[!INCLUDE [docs under construction](../includes/preview-note.md)]
+
+The `UserStoppedTypingBehavior` is a `Behavior` that allows the user to trigger an action when a user has stopped data input an `Entry`. Examples of its usage include triggering a search when a user has stopped entering their search query.
```

Comment:
Should we make this a bit more generic? Or at least point out that `Entry` is only one of the examples? Users might wrongfully think that this only works with an `Entry` while it also does with `SearchBar`, `Editor`, etc.

Additionally, I think the technical term is "debounce" for this. Should we mention that somewhere?

Reply:
Yes I copied this from the XCT but now you mention it I agree I will think of some new words.